src/asynk/async_queue.rs

```diff
@@ -617,17 +711,19 @@ mod async_queue_tests {
         assert_eq!(Some(2), number);
         assert_eq!(Some("AsyncTask"), type_task);
 
-        let result = AsyncQueue::<NoTls>::remove_tasks_type_query(&mut transaction, "mytype")
-            .await
-            .unwrap();
+        let result = test.remove_tasks_type("mytype").await.unwrap();
         assert_eq!(0, result);
 
-        let result = AsyncQueue::<NoTls>::remove_tasks_type_query(&mut transaction, "common")
-            .await
-            .unwrap();
+        let result = test.remove_tasks_type("common").await.unwrap();
         assert_eq!(2, result);
 
-        transaction.rollback().await.unwrap();
+        test.transaction.rollback().await.unwrap();
+    }
+
+    async fn insert_task(test: &mut AsyncQueueTest<'_>, task: &dyn AsyncRunnable) -> Task {
+        let metadata = serde_json::to_value(task).unwrap();
+        let task = test.insert_task(metadata, &task.task_type()).await.unwrap();
+        task
```

Comment:
the last two lines can be replaced with just
```rust
test.insert_task(metadata, &task.task_type()).await.unwrap()
```

Reply:
Done in [7bc47de](https://github.com/ayrat555/fang/pull/46/commits/7bc47dec117e3dc4d7957f48d5df0eb5185759ee)